central_web_server/handle_database.js

```diff
@@ -114,9 +113,45 @@ async function addSurveyResults(surveyData) {
     .insert()
     .into("survey_results")
     .setFieldsRows(surveyResults)
-    .toString()
-  await connection.query(surveyResultsQuery)
+    .toString();
+  await connection.query(surveyResultsQuery);
+  connection.end();
 }
 
+async function addSession(data) {
+  let connection = await mysql.createConnection(config.connection);
+  let sessionQuery = squel
+    .insert()
+    .into("session")
+    .setFieldsRows([data])
+    .toString();
+  let sessionResult = await connection.query(sessionQuery);
+  connection.end();
+  return sessionResult.insertId;
+}
 
-module.exports = { insertSpeciesData, getSpeciesLists, addSurveyResults };
+async function addLocation(data) {
+  console.log(data);
+  let connection = await mysql.createConnection(config.connection);
+  let locationQuery = squel
+    .insert()
+    .into("location")
+    .setFieldsRows(data)
+    .toString();
+  let locationResult = await connection.query(locationQuery);
+  connection.end();
+}
+
+async function getLocation() {
+  let connection = await mysql.createConnection(config.connection);
+  let getLocationQuery = squel
+    .select()
+    .from("location")
+    .toString();
+  let getLocationResult = await connection.query(getLocationQuery);
+  console.log(getLocationResult);
+  connection.end();
+}
+addLocation([{"name": "Mars"}]);
+getLocation();
```

Comment:
Remove the two lines above as they will create things every time the script is imported